src/Sylius/Bundle/UserBundle/Resources/config/serializer/Model.User.yml

```diff
@@ -20,7 +20,7 @@ Sylius\Component\User\Model\User:
     relations:
         - rel: user
```

Comment:
Ok, so we have a bug :smile: 

Proper fix - add route for api customer show
Quick fix - remove this `rel`

Are you willing to fix this?